cmd/gpu_plugin/fakedev/fake-8x-DG1.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fake-gen-config
+  namespace: validation
+immutable: false
+data:
+  fakedev.json: |-
+    {
```

Comment:
Should the base object be an array? In case we ever want to define a heterogenous fake system.

Reply:
If you mean different GPUs on different nodes, one could just run multiple fake plugins, restricted to different nodes.  Or code could be changed to read multiple JSON files, and if JSON file includes new include & exclude keys listing node names, that code compares against specified env var value.

If you mean different GPUs on same node, there could be a device offset key in JSON, with code changed to parse multiple config files.

Both of above changes would be fully backwards compatible i.e. they could be done later.

=> If needed, I'll add those features later on, but keep code simple for now.

PS. I do not think changing API later would be problem either, as this is just a (scalability) test tool, config API change only requires matching generator version, and one gets error warning if they do not match.